### DISCUSSION TITLE
Demo request

### DISCUSSION THREAD

**Discussion by Xyncgas** [hyperledger/cello#600] [Q&A] [answered]
It would be great if there is a demo website to try out various functionality

**Comment by yeasy** [comment]
Maybe some video is a good idea? @Xyncgas

**Comment by Xyncgas** [comment]
a bunch knowledge cards in a flex layout that I can scroll down to see them would be better than a video, since knowledge points inside video occurs linearly, but people can quickly glance at text and select anything important to see or send to friends

**Reply by yeasy** [reply, marked as answer]
After discussion, we do not have an appropriate and secure solution with vm resource to host a demo instance at this moment.
Would like to enhance the doc and tutorial instead.
